matrix_sdk_base/src/models/room_member.rs

```diff
@@ -42,7 +41,7 @@ pub struct RoomMember {
     /// If the user should be considered active.
     pub currently_active: Option<bool>,
     /// The unique id of the room.
-    pub room_id: Option<String>,
+    pub room_id: Option<RoomId>,
```

Comment:
Why is this still an `Option` if we're now explicitly passing a room id to the constructor?